orttraining/orttraining/core/graph/pipeline_transformer.cc

```diff
@@ -22,17 +22,20 @@ bool IsBackward(Node& node) {
   return (node.Description() == "Backward pass");
 }
 
+NodeArg& CreateInt64NodeArg(Graph& graph, std::string name) {
+  ONNX_NAMESPACE::TypeProto type_proto;
+  type_proto.mutable_tensor_type()->set_elem_type(ONNX_NAMESPACE::TensorProto_DataType_INT64);
+  auto actual_name = graph.GenerateNodeArgName(name);
+  auto& event = graph.GetOrCreateNodeArg(actual_name, &type_proto);
```

Comment:
>event [](start = 8, length = 5)

nit: seems the name is too specific for the function. Maybe CreateEventNodeArg for function name, or change this to node_arg.

Reply:
Changed it to `node_arg`. Thanks.

---
In reply to: [425968006](https://github.com/microsoft/onnxruntime/pull/3823#discussion_r425968006) [](ancestors = 425968006)